### ISSUE TITLE
Missing something (Sorry)

### ISSUE THREAD

**Issue by DonRichards** [Ricket/nodebot#26]
Step by step instructions seem incomplete and/or scattered. 

This is what I did
```terminal
$ git clone https://github.com/Ricket/nodebot/
$ cd nodebot
$ npm install
$ cp config.js.sample config.js
$ edit config.js
```

Edited the config.js by changing only
```js
global.nodebot_prefs = {
    realname: 'Don Richards',
    nickname: 'DonRichards',
    username: 'DonRichards',
    serverpass: null,
```

Then ran it
```terminal
$ node main.js
```

Added #islandora to the data/autojoin.txt file (with and without # symbol)

Keeps trying to connect to barjavel.freenode.net. Is this what it's suppose to do? I'm trying to access irc.freenode.net channel "Islandora". How do I accomplish this?

**Comment by Ricket** [comment]
There is a server key in the config file: https://github.com/Ricket/nodebot/blob/master/config.js.sample#L19

Did you by chance edit that? Make sure it is still set to irc.freenode.net.

FWIW, your step-by-step instructions are exactly what is in the README.md, in the same order, so I don't see what is scattered or incomplete...

**Comment by DonRichards** [comment]
It seemed incomplete because only changing what was explicitly expressed did not yield ideal results. Line #19 was still at irc.freenode.net

**Comment by Ricket** [comment]
Sorry for the trouble. I'm still not clear exactly what is wrong. Are you failing to connect? Are you able to connect if you use an IRC client on same machine you are trying to run nodebot on, to connect to irc.freenode.net:6667?

**Comment by DonRichards** [comment]
Found the missed instructions. I just added a PR to help. It was missing channels. Simple enough.

**Comment by DonRichards** [comment]
fixed in PR #27
